python/setup.py

```diff
@@ -125,16 +125,16 @@ def run(self):
         spark_dist = os.path.join(self.install_lib, "pyspark", "spark-distribution")
         rmtree(spark_dist, ignore_errors=True)
 
-        if ("HADOOP_VERSION" in os.environ) or ("HIVE_VERSION" in os.environ):
-            # Note that SPARK_VERSION environment is just a testing purpose.
-            # HIVE_VERSION environment variable is also internal for now in case
+        if ("PYSPARK_HADOOP_VERSION" in os.environ) or ("PYSPARK_HIVE_VERSION" in os.environ):
+            # Note that PYSPARK_VERSION environment is just a testing purpose.
```

Comment:
Why also change SPARK_VERSION -> PYSPARK_VERSION here? Is it related to PYSPARK_HADOOP_VERSION?

Reply:
`SPARK_VERSION` is just for testing purpose and I just simply thought it's good to make it consistent ;-).. the env `SPARK_VERSION` can be likely conflicted with something else anyway.

Reply:
As we use variable names `spark_version` in many places, is it good to change to `PYSPARK_VERSION` here?

Reply:
Oh, okay.

Reply:
I think it's okay. It's same for `hadoop_version` and `hive_version` too. I don't mind changing it back if you feel strongly. This is just for internal and testing purpose.